garden/comparison.md

```diff
@@ -322,6 +322,6 @@ Hydrodynamics | ✕  | ✓
 Ocean currents | ✕  | ✓
 Test fixture | ✓ | [✓](https://gazebosim.org/api/gazebo/6.6/test_fixture.html)
 Spherical coordinates | ✓ | ✓
-Generic comms system | ✕ | [✓](https://github.com/gazebosim/gz-sim/blob/ign-gazebo6/examples/worlds/perfect_comms.sdf)
+Generic comms system | ✕ | [✓](https://github.com/gazebosim/gz-sim/blob/gz-sim7/examples/worlds/perfect_comms.sdf)
```

Comment:
I took a look at the whole file to see what pattern it follows, and whether this line should be updated.
If we change this line, we'd have to change all the other lines with older versions, like line 55, 185, 211, 323, etc.

From those lines, it seems normal in this file for new features to link to an older version, which may have been the earliest version that a feature was made available in Ignition/Gazebo. In that case, we wouldn't want to update this, because it would then suggest that this feature wasn't available until Garden, when in fact it was already available in Fortress. 

So I would say this line and 85 probably shouldn't be changed.

Reply:
Will do.